<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>1054</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>POR FAVOR, PRENCHA OS CAMPOS ABAIXO</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DADOS INPUT</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <family>Roboto</family>
             </font>
            </property>
            <property name="text">
             <string>Interessado</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="input_interessado">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <family>Roboto</family>
             </font>
            </property>
            <property name="text">
             <string>Procolo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="input_protocolo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>1 - Área do Terreno</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_do_terreno">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>2 - Área anteriormente construido</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_anteriormente_construido">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>3 - Área computável a construir no subsolo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_computavel_a_construir_no_subsolo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>4 - Área não computável a construir no subsolo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_nao_computavel_a_construir_no_subsolo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>5 - Área computável a construir no pavimento térreo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_computavel_a_construir_terreo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>6 - Área não computável a construir no pavimento térreo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_nao_computavel_a_construir_terreo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>7 - Área computável a construir no pavimento superior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_computavel_a_construir_pavSup">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>8 - Área não computável a construir no pavimento superior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_nao_computavel_a_construir_pavSup">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_24">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>9 - Área não computável a construir no ático</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_nao_computavel_a_construir_atico">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>10 - Área livre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_livre">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_25">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>11 - Número de pavimentos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_numero_de_pavimento">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_28">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>12 - Altura total</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_altura_total">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_27">
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
        <property name="text">
         <string>22 - Área de cobertura</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_area_de_cobertura">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FERRAMENTAS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>SISTEMA DE GERENCIAMENTO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_calcular">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>CALCULAR ESTATISTICAS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_voltar">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>VOLTAR</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
